setup.py

```diff
@@ -35,14 +35,16 @@
 
 requirements = [
     'Flask>=0.10.1',
-    'six>=1.7.2',
     'invenio-access>=0.1.0',
     'invenio-accounts>=0.1.2',
     'invenio-base>=0.1.0',
+    'invenio-collections>=0.1.2',
```

Comment:
Only in tests?
Currently invenio-collections is imported by:
 - invenio_search/admin.py 
 - invenio_search/enhancers/collection_filter.py
 - invenio_search/facet_builders.py
 - invenio_search/forms.py
 - invenio_search/registry.py
 - invenio_search/utils.py
 - invenio_search/views/search.py